plugin.video.pcloud-video-streaming/resources/lib/test.py

```diff
@@ -1,10 +1,10 @@
-import pcloudapi
+from . import pcloudapi
```

Comment:
It's great that you have a test for your add-on, but please remove it from the repo submission.